### PR TITLE
Adds safe access to nodeType and nodeSecureUrl in getVideos

### PR DESCRIPTION
**Pull request by lautarovc**

### CODE REVIEW

File: __tests__/index.spec.ts [hunk 1]
```diff
@@ -235,6 +235,30 @@ describe(`#getLinkPreview()`, () => {
     expect(response.url).toEqual(`http://www.google.com/`);
     expect(response.mediaType).toEqual(`website`);
   });
+
+  it('should handle video tags without type or secure_url tags', async () => {
```
Comment (ospfranco) [inline __tests__/index.spec.ts:239]: 🎉

File: index.ts [hunk 2]
```diff
@@ -164,13 +164,13 @@ function getVideos(doc: cheerio.Root) {
       const node = nodes[index];
       if (node.type === `tag`) video = node.attribs.content;
 
-      nodeType = nodeTypes![index];
-      if (nodeType.type === `tag`) {
+      nodeType = nodeTypes && nodeTypes[index];
```
Comment (ospfranco) [inline index.ts:167]: Doesn't the optional operator work?

```suggestion
      nodeType = nodeTypes?.[index];
```

File: index.ts [hunk 3]
```diff
@@ -164,13 +164,13 @@ function getVideos(doc: cheerio.Root) {
...
+      if (nodeType?.type === `tag`) {
         videoType = nodeType ? nodeType.attribs.content : null;
       }
 
-      nodeSecureUrl = nodeSecureUrls![index];
-      if (nodeSecureUrl.type === `tag`) {
+      nodeSecureUrl = nodeSecureUrls && nodeSecureUrls[index];
```
Comment (ospfranco) [inline index.ts:172]: Same here

```suggestion
      nodeSecureUrl = nodeSecureUrls?.[index];
```

File: index.ts (hunk 2 above)
Comment (lautarovc) [reply index.ts:167]: Yeah it should work!

Comment (lautarovc) [reply index.ts:167]: Thanks!

**Comment by ospfranco** [comment]
@lautarovc hey, sorry it took me so long, it seems the title on one of the tests is wrong, would you mind taking a look afterwards I will merge it

**Pull request merged**